### ISSUE TITLE
Assembled sequence in FASTA format as input.

### ISSUE THREAD

**Issue by Pigrenok** [xjtu-omics/SVision#4]
Maybe I do not understand the internal works of the pipeline and it depends on reads, but is it possible to allow as input FASTA file with assembled contigs/scaffolds (as alternative to BAM file input)?

Or do you think it is better to align contigs/scaffolds/chromosomes to reference (not probably directly, but through simulated reads) and use this BAM file? Would you suggest preferred chain of tools to do that?

Thank you very much in advance for your help.

**Comment by jiadong324** [comment]
Thanks for your interests to our work.

> Maybe I do not understand the internal works of the pipeline and it depends on reads, but is it possible to allow as input FASTA file with assembled contigs/scaffolds (as alternative to BAM file input)?

SVision supports contigs as input.

> Or do you think it is better to align contigs/scaffolds/chromosomes to reference (not probably directly, but through simulated reads) and use this BAM file? Would you suggest preferred chain of tools to do that?

Yes, you can use minimap2 aligned contigs as input for SVision. I don't quite understand what do you mean by 'not probably directly, but through simulated reads'.

**Comment by Pigrenok** [comment]
How would SVision behave of three assembly is chromosome level assembly? I am concerned about alignment of whole chromosomes. I will probably try it and see what it will give me.

**Comment by jiadong324** [comment]
Yes, please try it. We only tested SVision on human assemblies, which was not chromosome level.
